api/src/main/java/org/openmrs/module/pihcore/PihCoreActivator.java

```diff
@@ -67,6 +68,7 @@ public void configureSystem() {
             else {
                 configurationSetup.configureConceptDependencies();
             }
+            DbEventSetup.setup();
```

Comment:
can we make this configurable by a runtime property, at least in the short-term, so we can phase this in and start using it on only one or two CI servers?

Reply:
Absolutely, good point.